.github/workflows/main.yml

```diff
@@ -20,21 +20,22 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node: [14]
+        node: [14, 16]
 
     steps:
       - name: Clone repository
         uses: actions/checkout@v2
         with:
           fetch-depth: 3
 
-      - name: Cache multiple paths
-        uses: actions/cache@v2
+      - uses: actions/setup-node@v2
```

Comment:
Previously, This test does not run Node.js 14, because GitHub Actions's default Node version is 16.

https://github.com/ghiscoding/lerna-lite/runs/5704182822?check_suite_focus=true

<img width="741" alt="image" src="https://user-images.githubusercontent.com/19714/160262681-27288b81-37b6-46e9-8a92-31cde486b13b.png">

I've added [setup-node](https://github.com/actions/setup-node) to be explicit.